pnpm-workspace.yaml

```diff
@@ -66,6 +66,7 @@ catalogs:
         "@khanacademy/wonder-blocks-progress-spinner": ^3.1.41
         "@khanacademy/wonder-blocks-search-field": ^5.1.53
         "@khanacademy/wonder-blocks-switch": ^3.3.24
+        "@khanacademy/wonder-blocks-tabs": ^0.3.32
```

Comment:
I think the latest version is 0.3.36?

https://khan.github.io/wonder-blocks/?path=/docs/packages-tabs-tabs--docs

<img width="519" height="99" alt="Screenshot 2025-11-20 at 11 23 45 AM" src="https://github.com/user-attachments/assets/36568537-4453-4ed2-a700-3598fe75fdd7" />